integration_tests/nativegraphics/src/test/java/org/robolectric/integrationtests/nativegraphics/HardwareAcceleratedActivityRenderTest.java

```diff
@@ -35,6 +38,13 @@ public void hardwareAcceleratedActivity_setup() throws Exception {
 
   @Test
   public void hardwareAcceleratedActivity_pixelCopy() throws Exception {
+    // This API is not supported correctly on macOS now.
+    assume()
+        .that(
+            Objects.requireNonNull(System.getProperty("os.name"))
+                .toLowerCase(Locale.US)
+                .contains("mac"))
```

Comment:
Isn't there a util/class in this test module to provide a common method to do this check?